libs/libtatsu/Makefile

```diff
@@ -0,0 +1,71 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libtatsu
+PKG_VERSION:=1.0.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://github.com/libimobiledevice/libtatsu/releases/download/$(PKG_VERSION)
+PKG_HASH:=4f69452d23e50e0ffbe844110e6ab6a900d080e051fbda3b7d595f679dee9bc5
+
+PKG_MAINTAINER:=Rosen Penev <rosenp@gmail.com>
```

Comment:
I doubt that @neheb knows about this

Reply:
Should I leave it as @neheb or change to what? He is currently set as the maintainer for all `libimobiledevice` packages, and on my previous PR, the new `libimobiledevice-glue` library was accepted.

Reply:
Thats up to him, you can not add new packages unless people are aware of it.

Reply:
Kind reminder, because usually it is not a good idea to add new packages and give maintainership to someone else.

Reply:
I don’t want maintainership. You can change it.

Reply:
@neheb @BKPepe 
Thanks for the follow up. Can you please advise what is the proper thing to do?
1. Assign myself as maintainer
2. Ask someone else
3. Leave maintainer empty
4. Other?

Reply:
@neheb @BKPepe 
I assigned myself as a maintainer for `libtatsu`. Please let me know if you prefer assigning someone else.